tests/serverspec/default.yml

```diff
@@ -11,3 +11,4 @@
       - hipchat
     apt_repo_to_add:
       - ppa:webupd8team/java
+    jenkins_master_ssh_passphrase: "passphrase"
```

Comment:
as i wrote elsewhere, the example playbook should be identical to `tests/serverspec/default.yml`. please update `README.md`.